### DISCUSSION TITLE
Plots with missing keys after Update!

### DISCUSSION THREAD

**Discussion by houmannourian** [Chia-Network/chia-blockchain#14246] [Support]
Hi,
30% of the existing plats had missing keys problems after updating to version 1.6.
This problem has been reported before, but I did not find a solution to fix the problem.
https://github.com/Chia-Network/chia-blockchain/discussions/10597?sort=old

In the debug file in the log path, the following message is displayed for missing plots.
Sample message for a plot with missing key:
“2022-12-29T07:59:02.930 harvester chia.plotting.manager : WARNING Plot E:\plot-k32-2021-07-22-10-03-313bd775cfb393aa9e00bd6e854a724ac16785456c2c2b15ad7c2ce5be8d0e1f.plot has a farmer public key that is not in the farmer’s pk list.”

Thanks for your support.

**Comment by MumfMeisterT** [comment]
First run `chia keys show` and take note of the farmer public key. You should also make sure that all the keys you expect to have are listed. then you should run `chia plots check`  and compare what the farmer keys shows to what you previously noted from chia keys show. 
What platform are you on & OS? 
What chia version did you have previously, and what version did you upgrade to?

**Reply by houmannourian** [reply]
Thank you for your response.
I am using Windows 10 LTSC version and I upgraded from 1.4 to 1.6.1.
I didn't understand how to do the said things.
Please help me to achieve the result step by step.
